edit-context/html-editor/index.html

```diff
@@ -475,25 +460,31 @@
         while (treeWalker.nextNode()) {
           const node = treeWalker.currentNode;
 
-          if (!anchorNode && offset + node.textContent.length >= start) {
+          if (!anchorNode && offset + node.textContent.length >= orderedStart) {
             anchorNode = node;
-            anchorOffset = start - offset;
+            anchorOffset = orderedStart - offset;
           }
 
-          if (offset + node.textContent.length >= end) {
+          if (offset + node.textContent.length >= orderedEnd) {
             extentNode = node;
-            extentOffset = end - offset;
+            extentOffset = orderedEnd - offset;
             break;
           }
 
           offset += node.textContent.length;
         }
 
-        return { anchorNode, anchorOffset, extentNode, extentOffset };
+        return {
+          anchorNode: isBackwards ? extentNode : anchorNode,
+          anchorOffset: isBackwards ? extentOffset : anchorOffset,
+          extentNode: isBackwards ? anchorNode : extentNode,
+          extentOffset: isBackwards ? anchorOffset : extentOffset
+        };
       }
 
       // Render the initial view.
       render();
+      setInterval(render, 1000);
```

Comment:
Can you explain why we have to do this now? It has the unfortunate effect of making it impossible to use the devtools inspector for the rendered text.